subsys/bluetooth/host_extensions/radio_notification_conn_cb.c

```diff
@@ -12,7 +12,11 @@
 
 #if defined(CONFIG_SOC_COMPATIBLE_NRF52X) || defined(CONFIG_SOC_COMPATIBLE_NRF5340_CPUNET)
 #include <hal/nrf_rtc.h>
-static uint32_t sys_clock_start_to_bt_clk_start_us;
+/* In case of use of nrf clock control MPSL is stared on POST_KERNEL level, hence NRF_RTC0 is
+ * started after system clock which is NRF_RTC1. In case MPLS is configured as clock control driver
+ * then NRF_RTC0 is started on PRE_KERNEL_1 init level.
```

Comment:
```suggestion
/* In case of use of nrf clock control MPSL is started on POST_KERNEL level, hence NRF_RTC0 is
 * started after system clock which is NRF_RTC1. In case MPSL is configured as clock control driver
 * then NRF_RTC0 is started on PRE_KERNEL_1 init level.
```